api/python/provisioner/api_spec.yaml

```diff
@@ -91,6 +91,8 @@ sw_update:
   type: SWUpdate
 fw_update:
   type: FWUpdate
+rollback_update:
```

Comment:
Should we change this to
**sw_rollback**

Reply:
addressed